app/views/declaration/summary/amend_summary_page.scala.html

```diff
@@ -16,43 +16,54 @@
 
 @import config.AppConfig
 @import controllers.declaration.routes
-@import forms.declaration.LegalDeclaration
-@import models.{ExportsDeclaration, Mode}
+@import models.Mode
 @import models.requests.JourneyRequest
-@import models.viewmodels.HtmlTableRow
+@import views.{BackButton, Title}
+@import views.html.components.gds.{linkButton, _}
 @import views.html.declaration.summary._
-@import views.{Title, ViewDates}
 
-@this(main_template: views.html.main_template, draftInfoSection: draft_info_section)
+@this(
+        govukLayout: gdsMainTemplate,
+        pageTitle: pageTitle,
+        paragraphBody: paragraphBody,
+        draftInfoSection: draft_info_section,
+        references_section: references_section,
+        countries_section: countries_section,
+        parties_section: parties_section,
+        locations_section: locations_section,
+        transaction_section: transaction_section,
+        items_section: items_section,
+        warehouse_section: warehouse_section,
+        transport_section: transport_section,
+        linkButton: linkButton
+)
 
 @()(implicit request: JourneyRequest[_], messages: Messages, appConfig: AppConfig)
 
-@main_template(title = Title("declaration.summary.amend-header")) {
+@govukLayout(
+ title = Title("declaration.summary.amend-header"),
+ backButton = Some(BackButton(messages("site.back"), controllers.routes.SubmissionsController.displayListOfSubmissions()))) {
 
-    @components.back_link(controllers.routes.SubmissionsController.displayListOfSubmissions(), Mode.Amend,  "summary.amend.back")
+ @pageTitle(messages("declaration.summary.amend-header"))
 
-    @components.heading(title = messages("declaration.summary.amend-header"))
+ @references_section(Mode.Draft, request.cacheModel)
```

Comment:
Why Mode has been changed to `Draft` from `ChangeAmend`? If this page is amend summary page.

Reply:
Good spot.  Copy/paste error